### ISSUE TITLE
Add graphite in BackendListener like InfluxDbBackendListener

### ISSUE THREAD

**Issue by Ioiss** [abstracta/jmeter-java-dsl#248]
Hello
Add graphite in BackendListener JMeter JAVA DSL.
I use graphite in my BackendListener on JMeter and don't have influx like data storage for grafana.

<img width="888" alt="graphiteBLJMeter" src="https://github.com/abstracta/jmeter-java-dsl/assets/38499256/8386af5d-0356-48f6-88b4-a355990a8872">

Kind regards!

**Comment by rabelenda** [comment]
Hello, thank you for asking this!

What do you think of something like this?

```java
graphiteListener(System.getenv("NODE_IP") + ":3003")
  .metricsPrefix("target.load-gen.jmeter.")
```

Do you usually change any other properties in the listener?

Regards

**Comment by Ioiss** [comment]
It's looks nice
Usually I use this properties.
Some times I write IP or Host graphite like: "127.0.0.1" or "http://test-graphite:8080"

**Comment by rabelenda** [comment]
Last week we released [1.25](https://github.com/abstracta/jmeter-java-dsl/releases/tag/v1.25) which includes `graphiteListener` element. [Here](https://abstracta.github.io/jmeter-java-dsl/guide/#graphite) is the new user guide section. 

I am closing this issue, but if you have further comments on this let us know and we can re open this ticket.
